### ISSUE TITLE
unable to activate homerow when there is no focus windows

### ISSUE THREAD

**Issue by frankiexyz** [dexterleng/homerow#43]
**Describe the bug**

unable to activate homerow when there is no focus windows/application, is it an expected behaviour?

**Comment by dexterleng** [comment]
Yup, that's how it works. I can make it work without a frontmost window though!

**Comment by yihou** [comment]
Yea, I think it would help if homerow still able to activate, so that users can quickly navigate the cursor to the header navigations.
